cachelib/shm/SysVShmSegment.cpp

```diff
@@ -184,6 +187,50 @@ void shmCtlImpl(int shmid, int cmd, shmid_ds* buf) {
   }
 }
 
+void mbindImpl(void *addr, unsigned long len, int mode,
+               const std::vector<size_t>& memBindNumaNodes,
+               unsigned int flags) {
+  struct bitmask *nodesMask = numa_allocate_nodemask();
```

Comment:
```suggestion
  auto nodesMask = numa_allocate_nodemask();
```

Reply:
And would we want to failfast on empty memBindNumaNodes like Posix?

Reply:
It is already done in `SysVShmSegment::memBind()`